typescript/ecs/cluster/index.ts

```diff
@@ -11,20 +10,25 @@ class ECSCluster extends cdk.Stack {
     const vpc = new ec2.Vpc(this, 'MyVpc', { maxAZs: 2 });
 
     const asg = new autoscaling.AutoScalingGroup(this, 'MyFleet', {
-      instanceType: new InstanceType("t2.xlarge"),
+      instanceType: new ec2.InstanceType(ec2.InstanceSize.MICRO),
```

Comment:
Does this work? You're replacing an instance type with instance size? 't2.xlarge' with 'micro'

Reply:
While it's not within the scope of the issue this PR addresses (0.36.0 update), I think this is a good change to make. These are samples for people to learn on and, as such, should minimise use of more expensive resources.

Reply:
I think what they mean is that there’s a difference between an instance type t2, m5, c5, etc..... and instance sizes micro, medium, 16xLarge, etc.....

This code change has replaced a type with a size and it may not work

Reply:
Ah yes, you're right.
It can take a string (e.g. 't2.micro') or there's the 'of' method which takes an InstanceClass and InstanceSize.
[ec2.InstanceType TypeScript reference](https://docs.aws.amazon.com/cdk/api/latest/typescript/api/aws-ec2/instancetype.html#aws_ec2_InstanceType)

Reply:
@pkandasamy91 you are correct it should be as it was. However, I would argue that `InstanceType` constructer should also take an enumeration combination of class and type. 

Like this:
`new InstanceType(ec2.InstanceClass.T2, ec2.InstanceSize.XLARGE)`

It make it more strongly type. But for now I'll update it back to what it was.

Reply:
commit c0937290b1474e67b5686fc4845a07b628ce9bef addresses this issue across all files.